### ISSUE TITLE
Genbank-output: circular-tag

### ISSUE THREAD

**Issue by xJane01x** [nigyta/dfast_core#45]
Hallo, 

very useful and fast annotation tool!

How can I add the circular tag for complete chromosomes and plasmids in the genbank file  instead of editing manually? Is there any option?

completness and topology are part of the fasta-identifier in the input files.

Thanks for your help.

**Comment by nigyta** [comment]
Yes, you can add them with `--complete t`,  `--seq_names`, `--seq_topologies`, `--seq_types` options.

Here is a minimum example:
```
dfast -g example/sample.lactobacillus.fna --complete t  --seq_names "Chromosome,pXXXX,pYYYY" --seq_topologies c,c,l --seq_types c,p,p --no_cdd --no_hmm --no_cds --no_rrna --no_trna --no_crispr --force
```
In this example, the input genome consists of 1 circular chromosome (Chromosome), 1 circular plasmid (pXXXX), 1 linear plasmid (pYYY).

Please see the document below for more detail:
https://github.com/nigyta/dfast_core/blob/master/docs/insdc_submission.md#for-a-complete-genome
